functional_tests/test_db_online_offline.py

```diff
@@ -18,7 +19,7 @@
 import logging
 log = logging.getLogger(lib.settings.LOGGER)
 
-NUM_ENDPOINTS = 10
+NUM_ENDPOINTS = 13
 
 def rest_scan(sync_gateway, db, online, num_docs, user_name, channels):
```

Comment:
I'm not really sure what you mean by this. Can you please elaborate? There is some persistent state between test runs, however, it's pretty tied to these scenarios and should not be used like a library function.